tests/acceptance/helpers/webdavHelper.js

```diff
@@ -221,8 +221,8 @@ exports.getSkeletonFile = function (filename) {
 
 exports.uploadFileWithContent = function (user, content, filename) {
   const headers = httpHelper.createAuthHeader(user)
-  return fetch(client.globals.backend_url +
-    '/remote.php/webdav/' + filename, {
+  return fetch(join(client.globals.backend_url,
+    '/remote.php/webdav/', filename), {
```

Comment:
Is this too large to separate into two lines?

Reply:
fixed